### PR TITLE
Expand ssl_assert_fingerprint docs

### PR DESCRIPTION
**Pull request by zmoog**
## What does this PR do?

Expand the docs about the `ssl_assert_fingerprint` configuration option adding:

- More details about its default value and related behavior 
- A new section that explains what the option is and how to set it up.

## Why is it important?

A few users struggled to set it up and created support requests, so this docs update aims to make using clusters with self-signed certificates as smooth as possible.

## Checklist

- ~~[ ] My code follows the style guidelines of this project~~
- ~~[ ] I have commented my code, particularly in hard-to-understand areas~~
- ~~[ ] I have made corresponding change to the default configuration files~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added an entry in `CHANGELOG.md`~~
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by aspacca — APPROVED** [review]
🥇

File: docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc [hunk 1]
```diff
@@ -234,7 +234,7 @@ For `logstash` the following arguments are supported:
   * `args.password`: (Optional) Password of the {ls} instance to connect to. Mandatory if HTTP Basic authentication is enabled in {ls}.
   * `args.max_batch_size`: (Optional) Maximum number of events to send in a single HTTP(s) request. Default value: 500
   * `args.compression_level`: (Optional) The GZIP compression level for HTTP(s) requests towards {ls}. It can be any integer value between 1 (minimum compression, best performance, highest amount of bytes sent) and 9 (maximum compression, worst performance, lowest amount of bytes sent). Default value: 1
-  * `args.ssl_assert_fingerprint`: (Optional) SSL fingerprint for self-signed SSL certificate on HTTPS transport.
+  * `args.ssl_assert_fingerprint`: (Optional) SSL fingerprint for self-signed SSL certificate on HTTPS transport. The default value is an empty string, meaning the HTTP client requires a valid certificate.
```
Comment (girodav) [inline docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc:237]: NIT: I wonder if we should explain what "valid" means here

**Review by girodav — APPROVED** [review]

File: docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc (hunk 1 above)
Comment (zmoog) [reply docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc:237]: Good point. 

What about this updated description?

> SSL fingerprint for self-signed SSL/TLS certificate on HTTPS transport. The default value is an empty string, meaning the HTTP client requires a valid certificate (issued by a trusted CA, valid date range, matching CN, and valid certificate chain).

Comment (girodav) [reply docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc:237]: That is definitely enough :)

**Pull request merged**